### ISSUE TITLE
Completed and Incomplete Tasks have 2 different section.

### ISSUE THREAD

**Issue by purv2100** [aditicsalunkhe/Simplii-Group74-2023#43]
**Is your feature request related to a problem? Please describe.**
At first, when you completed the task, the task did not go anywhere it just stayed right there. The frontend had no solution even though in the backend logic was implemented.

**Describe the solution you'd like**
Completed task would have a separate section than incomplete tasks.

**Comment by aditicsalunkhe** [comment]
Yes, it will contribute greatly towards improving user-experience. Thank you so much!

**Comment by WangWangIsDog** [comment]
A great improvement!
Also be careful to implement this if this has some impact to Analytics section

**Comment by purv2100** [comment]
Yes, I need to work on that keeping analytics section in mind.
